src/utils.ts

```diff
@@ -1,21 +1,47 @@
 import { Dispatch } from "./dispatch";
 
 export class Util {
-    async getAllPages(path: string, arrayName: string, dispatch: Dispatch) {
+    static async getAllPages(path: string, arrayName: string, dispatch: Dispatch, filter?: string) {
         const items: any = await dispatch.get(path);
         
+        if (filter)
+            path += '?filter=' + filter;
+
         let allItems: any[] = [];
         const totalPages = Number.parseInt(items.MetaInformation['@TotalPages']);
         allItems.push(...items[arrayName]);
         
         if (totalPages > 1){
             let currentPage: number = 2;
             while(currentPage <= totalPages) {
-                allItems.push(...await dispatch.get(path + '&page=' + currentPage)[arrayName]);
+                const pageItems: any = await dispatch.get(path + (filter ? '&' : '?') + 'page=' + currentPage);
+                allItems.push(...pageItems[arrayName]);
                 currentPage++;
             }
         }
         
         return allItems;
     }
+
+    // convert fortnox error into standard error
+    static async makeErrorFromResponse (response: any) {
+
+        // get fortnox error if exist
+        if (response?.headers?.get('content-type')?.startsWith('application/json')) {
+            const fnError = await response.json();

```

Comment:
Fortnox error is packed like `{ ErrorInformation: { message: "bla-bla-bla", code: 100500 }`